src/Mimi.jl

```diff
@@ -14,50 +14,32 @@ using StringBuilders
 
 export
     @defcomp,
-    @modelegate,
     MarginalModel,
     Model,
-    addcomponent,
-    add_connector_comps,
-    add_dimension!,
-    addparameter,
-    compdef,
-    compdefs,
-    compinstance,
-    compkeys,
+    addcomponent,  # push!, add_component!, add_component, addcomp!, add!; could also have multiple functions like insert!, push! etc. instead of before after keywords
+    add_connector_comps, #should this be deleted?  Docs indicate we need it ...
     components,
-    connect_parameter,
+    connect_parameter, # connect!, bind, assign, link; ! yes or no; frontrunner: connect!
     create_marginal_model,
-    datatype,
-    description,
-    dimension,
-    dimensions,
-    disconnect!,
+    disconnect!, # disconnect_parameter!; frontrunner: disconnect!
```

Comment:
suggestion:  `disconnect_param!`
reason:  matches the above `connect_param` but uses `!` since it alters an existing connection

Reply:
Agreed.